.github/workflows/preview.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     with:
       sha: ${{ github.event.pull_request.head.sha }}
       environment: staging
-      netlify-context: deploy-preview
+      netlify-context: branch-deploy
```

Comment:
We were actually using what Netllify calls branch deploys. (It doesn't seem to  make any difference)